Sources/EdgeNetworkHandlers/EdgeEventError.swift

```diff
@@ -31,6 +31,26 @@ struct EdgeEventError: Codable, Equatable {
 
     /// A report for the error containing additional information
     let report: EdgeErrorReport?
+
+    // MARK: - Codable
+    enum CodingKeys: String, CodingKey {
+        case title
+        case detail
+        case status
+        case type
+        case report
+        case eventIndex
+    }
+
+    public func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: CodingKeys.self)
+
```

Comment:
let's add a comment, something like // skip eventIndex 
same for the other structs